qiskit_metal/renderers/renderer_ansys/ansys_renderer.py

```diff
@@ -162,42 +195,120 @@ class QAnsysRenderer(QRenderer):
         resistance=default_options['_Rj'],
         mesh_kw_jj=parse_units(default_options['max_mesh_length_jj'])))
 
-    def __init__(self,
-                 design: 'QDesign',
-                 initiate=True,
-                 render_template: Dict = None,
-                 render_options: Dict = None):
+    def __init__(self, design: 'QDesign', initiate=True, options: Dict = None):
         """
         Create a QRenderer for Ansys.
 
         Args:
             design (QDesign): Use QGeometry within QDesign to obtain elements for Ansys.
             initiate (bool, optional): True to initiate the renderer. Defaults to True.
-            render_template (Dict, optional): Typically used by GUI for template options for GDS. Defaults to None.
-            render_options (Dict, optional):  Used to override all options. Defaults to None.
+            options (Dict, optional):  Used to override all options. Defaults to None.
         """
-        super().__init__(design=design,
-                         initiate=initiate,
-                         render_template=render_template,
-                         render_options=render_options)
+        super().__init__(design=design, initiate=initiate, options=options)
 
         # Default behavior is to render all components unless a strict subset was chosen
         self.render_everything = True
 
         self._pinfo = None
+        # Conected to Ansys variables
+        self._rapp = None
+        self._rdesktop = None
+
+    @property
+    def initialized(self):
+        if self._pinfo:
+            if self._pinfo.project:
+                return True
+        return False
```

Comment:
`return self._pinfo and self._pinfo.project`

Reply:
Cannot do that because it will cause a AttributeError if self._pinfo = None